packages/dialect-react-ui/components/NotificationCenterButton/index.tsx

```diff
@@ -111,25 +76,9 @@ function WrappedNotificationCenterButton(
           // className="w-full h-full bg-white/10"
           // style={{ backdropFilter: 'blur(132px)' }}
         >
-          <NotificationCenter />
+          <NotificationCenter className={popup} />
```

Comment:
what's the purpose of this `className`? how does it relate to the themes variables in the provider?

Reply:
I just moved the className for notification center popup from inside to better control the styling